src/stores/Runbooks.js

```diff
@@ -0,0 +1,30 @@
+import {observable} from 'mobx';
+import {uniq} from 'lodash';
+
+export class Runbook {
+  @observable _id = null
```

Comment:
Probably this attribute should be renamed and become not observable - it won't make sense to change id